[2023-01-17T22:37:45.605+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: accidents_data_etl_pipeline.extract_integrate_dataset manual__2023-01-17T22:37:39.465011+00:00 [queued]>
[2023-01-17T22:37:45.641+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: accidents_data_etl_pipeline.extract_integrate_dataset manual__2023-01-17T22:37:39.465011+00:00 [queued]>
[2023-01-17T22:37:45.643+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-01-17T22:37:45.650+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 2
[2023-01-17T22:37:45.652+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-01-17T22:37:45.725+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): extract_integrate_dataset> on 2023-01-17 22:37:39.465011+00:00
[2023-01-17T22:37:45.740+0000] {standard_task_runner.py:55} INFO - Started process 9249 to run task
[2023-01-17T22:37:45.746+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'accidents_data_etl_pipeline', 'extract_integrate_dataset', 'manual__2023-01-17T22:37:39.465011+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpv5m7tnr1']
[2023-01-17T22:37:45.749+0000] {standard_task_runner.py:83} INFO - Job 42: Subtask extract_integrate_dataset
[2023-01-17T22:37:45.920+0000] {task_command.py:389} INFO - Running <TaskInstance: accidents_data_etl_pipeline.extract_integrate_dataset manual__2023-01-17T22:37:39.465011+00:00 [running]> on host 7fc7752d3cb4
[2023-01-17T22:37:46.071+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=accidents_data_etl_pipeline
AIRFLOW_CTX_TASK_ID=extract_integrate_dataset
AIRFLOW_CTX_EXECUTION_DATE=2023-01-17T22:37:39.465011+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-17T22:37:39.465011+00:00
[2023-01-17T22:37:46.078+0000] {logging_mixin.py:137} INFO - got file paths successfully
[2023-01-17T22:37:50.390+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/***/operators/python.py:192 DtypeWarning: Columns (10) have mixed types.Specify dtype option on import or set low_memory=False.
[2023-01-17T22:37:53.886+0000] {logging_mixin.py:137} INFO - loaded CSVs files to df successfully
[2023-01-17T22:38:26.238+0000] {logging_mixin.py:137} INFO - data integrated successfully
[2023-01-17T22:38:26.365+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-01-17T22:38:26.689+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=accidents_data_etl_pipeline, task_id=extract_integrate_dataset, execution_date=20230117T223739, start_date=20230117T223745, end_date=20230117T223826
[2023-01-17T22:38:27.005+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-01-17T22:38:27.280+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
